index.php

```diff
@@ -19,9 +52,9 @@
  *     production
  *
  * NOTE: If you change these, also change the error_reporting() code below
- *
  */
-define('ENVIRONMENT', 'development');
+	define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'development');
```

Comment:
(Note this change comes from the upstream stock index.php that I'm rebasing onto, rather than something I wrote.) It'd be set by the webserver; see [examples here](https://stackoverflow.com/questions/36051146/codeigniter3-why-would-serverci-env-ever-be-set-in-the-first-place) that fills in for missing docs.